File: Clang/Basics/simple_intrest.c
#include <stdio.h>
int main()
{
	float p, r, t;
	printf ("Enter principal value: ");
	scanf ("%f", &p);
	printf ("Enter rate of interest: ");
	scanf ("%f", &r);
	printf ("Enter time in years: ");
	scanf ("%f", &t);
	printf ("SI=%f", p*r*t/100);
	
}

File: Python/Basics/simple_intrest.py
# WAP in python to print Simple intrest for a given p, r and t.

p = int( input("Enter principal value: "))
r = int( input("Enter rate of intrest: "))
t = int( input("Enter the time period: "))

si = p*r*t/100
print(f"calculated intrest ={si}.")
